docs/modular/ansible/index.md

```diff
@@ -218,14 +218,54 @@ The main grid view uses the bottom row for navigation, which are dimly lit into
 
 ![](images/grid_KR_NAVIGATION_1.2.png)
 
-Each *track* has parameters *Trigger, Note, Octave,* and *Duration*. These parameters comprise of the sequence loop.
+### Parameters
+
+Each *track* has four parameters *Trigger, Note, Octave,* and *Duration*. These parameters work together to form the sequence loop.
 
 A *note* will occur only when a *trigger* is set. This note will be the pitch specified by *note* in *octave* of a specified *duration*.
 
 The *trigger* view shows all four tracks at once, whereas all other parameter views display the track selected.
 
 The *duration* view has downward sliders (lower is longer) and a master duration multiplier at the top.
 
+#### Parameter extensions
+
+Many of the parameter views have sub pages that extend or augment their functionality. To enter into each sub page, switch to the parameter using buttons 5-8, then tap the parameter button again. When you are in the parameter extension page, the parameter button will blink, to assist with navigation.
+
+All of the subpages have their own probability, clock division and loop controls, allowing for some very complex phrasing to be built.
+
+* *Trigger* view becomes *Trigger Ratcheting*
+* *Note* view becomes *Alternate Note*
+* *Octave* view becomes *Glide*
```

Comment:
typo? GLide

Reply:
No I think that is just a lowercase 'l' doing its best not to get confused for an I or a 1

Reply:
Ah, with my glasses on it's a lower 'l' in italics that was confusing me!!